src/layouts/CityForecast/Container.jsx

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { THEMES, BG_IMAGES } from '../../constants/themes';
+import { CityForecastTypes, CityInfoTypes } from '../../types/WeatherDataTypes';
+import CityForecast from './CityForecast';
+
+function CityForecastContainer({ cityForecast, cityInfo, theme }) {
+  const themeBg = theme === THEMES.light ? BG_IMAGES.light : BG_IMAGES.dark;
```

Comment:
I'd make it a part of inner component in order to keep Container just a wrapper over the CityForecast without any additional logic. You can use selector for that matter. Plus, themeBg - is it a single source property always applicable for all components across the app? If yes - it can be shared from a higher level